banzai/tic-tac-toe.rb

```diff
@@ -0,0 +1,108 @@
+# this gem injects a color method to the string when it is displayed by console
+require 'colorize' 
+
+@level = 2
+
+def draw_field(val, index)
+  (val.is_a? Integer) ? (index + 1 < 10 ? " #{val}" : val) : " #{val}"
```

Comment:
This could have been a normal `if`, this is clever but it harder to read, don't try to be clever, remember you will spend more time reading code than writing it, optimize for readability 😉